src/quick-lint-js/error.h

```diff
@@ -1188,6 +1188,13 @@
           QLJS_TRANSLATABLE("return statement returns nothing (undefined)"),   \
           return_keyword))                                                     \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_statement_before_first_switch_case, "E0198",                       \
+      { source_code_span unexpected_statement; },                              \
+      ERROR(QLJS_TRANSLATABLE("unexpected statement before first switch "      \
+                              "case, expected 'case' or 'default'"),           \
```

Comment:
Nit: write a semicolon instead of a comma:
```suggestion
                              "case; expected 'case' or 'default'"),           \
```